### ISSUE TITLE
Bondless Takers

### ISSUE THREAD

**Issue by jacobranson** [RoboSats/robosats#871]
I know there are mentions of bondless takers sprinkled throughout other issues and on the RoboSats learn site, but I did not see an issue tracking the progress of this work, and what specifically needs to be done to make this happen.

**Comment by Reckless-Satoshi** [comment]
> I know there are mentions of bondless takers sprinkled throughout other issues and on the RoboSats learn site, but I did not see an issue tracking the progress of this work, and what specifically needs to be done to make this happen.

Indeed, it has been discussed previously, even, there was API parameters for bondless offers. However, this issue has not seen any progress as it is rather clear that accepting bondless takers is unsafe. So we might need to find a better system for new users that do not have any Sats to get started. 

Maybe something like a "buddy program": a chat group where some experienced users can volutarily offer to lock up small bonds for new users. Thoughts on this?

**Comment by jacobranson** [comment]
I think that's a great idea. Some onboarding ramp for new users would be ideal. Maybe a SimpleX chat room would be a great solution for something like this.
